src/Serialization/WitnessSet.js

```diff
@@ -0,0 +1,66 @@
+/* global require exports */
+
+var lib;
+if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
+    lib = require('@ngua/cardano-serialization-lib-browser');
+} else {
+    lib = require('@ngua/cardano-serialization-lib-nodejs');
+}
```

Comment:
Don't have much JS experience but we have this code repeated in a lot of js modules. Should we centralise this?